<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayDockWidgetBase</class>
 <widget class="QDockWidget" name="DisplayDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="logPlotGroupBox">
      <property name="title">
       <string>Log plot</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QFormLayout" name="formLayout_8">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QSlider" name="logPlotBaseSlider">
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="logPlotBaseLineEdit">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
           </property>
           <property name="text">
            <string>10</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Base:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Scale</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="planeOfIncidenceCheckBox">
         <property name="text">
          <string>Show scales in the plane of incidence</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Radius 1:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="radius1SpinBox">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>9999.989999999999782</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Radius 2:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="radius2SpinBox">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>9999.989999999999782</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>planeOfIncidenceCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>DisplayDockWidgetBase</receiver>
   <slot>showScaleInPlaneOfIncidence(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radius1SpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DisplayDockWidgetBase</receiver>
   <slot>setScaleLength1(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radius2SpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DisplayDockWidgetBase</receiver>
   <slot>setScaleLength2(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logPlotGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>DisplayDockWidgetBase</receiver>
   <slot>useLogPlot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logPlotBaseSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DisplayDockWidgetBase</receiver>
   <slot>updateBaseOfLogarithm(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showScaleInPlaneOfIncidence(bool)</slot>
  <slot>setScaleLength1(double)</slot>
  <slot>setScaleLength2(double)</slot>
  <slot>useLogPlot(bool)</slot>
  <slot>updateBaseOfLogarithm(int)</slot>
 </slots>
</ui>
